### PR TITLE
Feature/pt 203

### PR DESCRIPTION
**Pull request by luktar**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+  "version": "0.2.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.4.1",
+    "@emotion/styled": "^11.3.0",
     "@material-ui/core": "^4.12.3",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.60",
+    "@mui/icons-material": "^5.0.0",
+    "@mui/material": "^5.0.0",
```
Comment (AdamSarach) [inline package.json:12]: Do we really want to keep 2 different version of material (4.x vs 5.x) ?

File: src/components/MapManager.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    setPropertyGrid(newGrid);
+  }
+
+  return (
+    <div className="map-manager">
+      <div className="map-manager__sidebar">
+        <DashboardSidebar
+          isLoggedIn={isLoggedIn}
+          propertyGrid={propertyGrid}
+          updatePropertyGrid={updatePropertyGrid} />
```
Comment (AdamSarach) [inline src/components/MapManager.jsx:26]:
```suggestion
          updatePropertyGrid={setPropertyGrid} />
```

File: src/components/MapManager.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+import '../stylesheets/mapManager.scss';
+
+const MapManager = () => {
+
+  const [propertyGrid, setPropertyGrid] = useState(null);
+
+  const isLoggedIn = useSelector(selectAuthIsLoggedIn);
+
+  function updatePropertyGrid(newGrid) {
```
Comment (AdamSarach) [inline src/components/MapManager.jsx:16]: To be deleted when suggestion implemented. I know its my piece of code by the way  :P

File: src/components/MapView.jsx [hunk 4]
```diff
@@ -1,37 +1,37 @@
...
 
 import api from '../api/apiInstance';
-import {basicHeaders} from '../config/apiConfig';
-import {NotificationActions} from '../redux/actions/notificationActions';
-import {unsafeApiError} from '../utilities/utilities';
-import {MapContext} from './contexts/MapContextProvider';
+import { basicHeaders } from '../config/apiConfig';
+import { NotificationActions } from '../redux/actions/notificationActions';
+import { unsafeApiError } from '../utilities/utilities';
+import { MapContext } from './contexts/MapContextProvider';
```
Comment (AdamSarach) [inline src/components/MapView.jsx:11]: In our `.prettierrc` file there is property `"bracketSpacing": false` - please check if you use proper formatter in your VSCode -> Ctrl+Shift+P -> `Format Document with...`, use prettier or run command in terminal `npm run prettier:components`

File: src/components/MapView.jsx [hunk 5]
```diff
@@ -1,37 +1,37 @@
...
+  const { t } = useTranslation();
   const [allStops, setAllStops] = useState([]);
   const [activeBusStopId, setActiveBusStopId] = useState(null);
   const dispatch = useDispatch();
   const authRoles = useSelector(selectLoggedInUserRoles);
 
-  const currentLocation = {lat: 50.29, lng: 19.01};
+  const currentLocation = { lat: 50.29, lng: 19.01 };
   const zoom = 10;
   const maxZoom = 19;
```
Comment (AdamSarach) [inline src/components/MapView.jsx:30]: It would be awesome to have these constants in separate file or env file

File: src/components/Footer.jsx [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+const Footer = () => {
+
+  return (
+    <div className="footer">
+      <div className="footer__left">
+        <VersionLabel />
+      </div>
+      <div className="footer__center">
+      Copyright © 2021 | rozwiazaniadlaniewidomych.org | All Rights Reserved
```
Comment (AdamSarach) [inline src/components/Footer.jsx:13]: :+1:

File: src/components/managementPanel/ManagementPanel.jsx [hunk 7]
```diff
@@ -1,32 +1,32 @@
...
 import H4Title from '../customs/H4Title';
-import {NotificationActions} from '../../redux/actions/notificationActions';
+import { NotificationActions } from '../../redux/actions/notificationActions';
 
 import '../../stylesheets/managementPanel.scss';
 
 const NONE = 'none';
-const CURRENT_LOCATION = {lat: 50.29, lng: 19.01};
+const CURRENT_LOCATION = { lat: 50.29, lng: 19.01 };
 const ZOOM = 9;
```
Comment (AdamSarach) [inline src/components/managementPanel/ManagementPanel.jsx:26]: as above, let's push it into separate file for constants (we do not have any at present)

File: src/config/apiConfig.js [hunk 8]
```diff
@@ -3,6 +3,7 @@ const noTokenHeaders = () => {
     'Cache-Control': 'no-cache',
     'Content-Type': 'application/json',
     'Access-Control-Allow-Origin': '*',
+    'Accept-Language': 'pl-pl'
```
Comment (AdamSarach) [inline src/config/apiConfig.js:6]: :+1:

File: src/stylesheets/dashboard.scss [hunk 9]
```diff
@@ -1,89 +1,24 @@
...
+  right: 0;
+  bottom: 0;
 
-.sidebar .nav-link:hover .feather,
-.sidebar .nav-link.active .feather {
-  color: inherit;
-}
+  display: grid;
+  grid-template-areas: "header" "content" "footer";
+  grid-template-rows: 3.5rem auto 1.5rem;
```
Comment (AdamSarach) [inline src/stylesheets/dashboard.scss:10]: great!

**Review by AdamSarach — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (luktar) [reply package.json:12]: It's just for icons ;)

File: src/components/MapView.jsx (hunk 4 above)
Comment (luktar) [reply src/components/MapView.jsx:11]: Ok, I'll check that next time.

**Pull request merged**